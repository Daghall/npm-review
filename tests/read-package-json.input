{
  "dependencies": {
    "express": "^4.17.1",
    "uuid": "npm:uuid/v4@0.1.0"
  },
  "devDependencies": {
    "mocha": "~9.0.0",
    "nodemon": "^1.2.3"
  }
}
