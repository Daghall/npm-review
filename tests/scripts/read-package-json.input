{
  "dependencies": {
    "angel": "^5.1.1",
    "buffy": "^4.1.1",
    "doc-brown": "^4.2.0",
    "jayne": "^2.4.0",
    "mal": "^1.0.0",
    "marty": "npm:marty/mcfly@5.1.0",
    "river": "^5.0.0",
    "spike": "^4.1.2",
    "willow": "^1.2.0"
  },
  "devDependencies": {
    "ang": "~2.3.0",
    "katara": "^4.1.3",
    "lt-cmdr-data": "~3.7.0",
    "picard": "~3.0.0",
    "riker": "~3.1.0",
    "sokka": "~3.5.0"
  }
}
