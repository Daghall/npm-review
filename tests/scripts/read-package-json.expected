angel	5.1.1	.dependencies
buffy	4.1.1	.dependencies
doc-brown	4.2.0	.dependencies
jayne	2.4.0	.dependencies
mal	1.0.0	.dependencies
marty	5.1.0	.dependencies	marty/mcfly
river	5.0.0	.dependencies
spike	4.1.2	.dependencies
willow	1.2.0	.dependencies
ang	2.3.0	.devDependencies
katara	4.1.3	.devDependencies
lt-cmdr-data	3.7.0	.devDependencies
picard	3.0.0	.devDependencies
riker	3.1.0	.devDependencies
sokka	3.5.0	.devDependencies
